client/multi-currency/enabled-currencies-list/modal-checkbox.js

```diff
@@ -36,7 +36,8 @@ const EnabledCurrenciesModalCheckbox = ( {
 						name: <span>{ name }</span>,
 						code: (
 							<span className="enabled-currency-checkbox__code">
-								({ symbol } { code })
+								({ symbol }
+								{ symbol === code ? null : ' ' + code })
```

Comment:
Little nitpick here, you can skip it 🙂 
This line could be simplified using `&&`, because JSX will not show `false`, only the truthy path.

```suggestion
								({ symbol }
								{ symbol !== code && ' ' + code })
```

Reply:
Sure, changed it.